tsconfig.json

```diff
@@ -2,12 +2,14 @@
   "extends": "@stellar/tsconfig",
   "compilerOptions": {
     // temporary flags during js->ts transition
-    "allowJs": false, 
+    "allowJs": true, 
     "isolatedModules": false,
-
-    "target": "es5",      
-    "rootDir": "./backend",
-    "baseUrl": "./"
+    "target": "es6",
```

Comment:
changing target to es6 because of [sequelize](https://github.com/sequelize/sequelize/issues/7840#issuecomment-317212324)